src/libraries/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -359,6 +359,7 @@ public static IEnumerable<object[]> InvalidPrefix_TestData()
 
         [Theory]
         [MemberData(nameof(InvalidPrefix_TestData))]
+        [SkipOnPlatform(TestPlatforms.FreeBSD, "FreeBSD accepts some as valid and test hangs")]
```

Comment:
"accepts some as valid"... some what?  I'm not understanding the comment.

Reply:
some items from `InvalidPrefix_TestData`. `http://192./` for example. I did not dig deeper why that is happening.